torch/_meta_registrations.py

```diff
@@ -2575,6 +2575,11 @@ def mkldnn_rnn_layer_backward(
     return diff_x, diff_w1, diff_w2, diff_b, diff_b, diff_hx, diff_cx
 
 
+@register_meta(aten.bucketize)
+def meta_bucketize(a, boundaries, *, out_int32=False, right=False):
+    return a
```

Comment:
`return a` if only the aten op is an inplace op.
Otherwise, you need to `return torch.empty_like(self)`

Reply:
as meta function need to correctly propagate "aliasing relationship"